code/game/antagonist/antagonist.dm

```diff
@@ -136,13 +146,23 @@
 	// Keeping broken up for readability
 	for(var/datum/mind/player in mode.get_players_for_role(id))
 		if(ghosts_only && !(isghostmind(player) || isnewplayer(player.current)))
-		else if(config.use_age_restriction_for_antags && player.current.client.player_age < minimum_player_age)
-		else if(player.special_role)
-		else if (player in pending_antagonists)
-		else if(!can_become_antag(player))
-		else if(player_is_antag(player))
-		else
-			candidates |= player
+			continue
+		if(config.use_age_restriction_for_antags)
+			if(!isnum(player.current.client.player_age))
+				log_warning("Config option USE_AGE_RESTRICTION_FOR_ANTAGS is set but a database connection could not be established")
+				continue
+			else if(player.current.client.player_age < minimum_player_age)
+				continue
+		if(player.special_role)
+			continue
+		if(player in pending_antagonists)
+			continue
+		if(!can_become_antag(player))
+			continue
+		if(player_is_antag(player))
+			continue
```

Comment:
Could have just OR'd these conditions instead (`if ((player.special_role) || (player in pending_antagonists) || (!can_become_antag(player)) || (player_is_antag(player))) continue`)

Reply:
Yep! Though the commented code above said they'd broke it up for better readability so I decided to leave it as it /shrug